dom-calculator-starter-code/js/calc.js

```diff
@@ -0,0 +1,71 @@
+//make variables from class/id
+
+var buttonNum = document.getElementsByClassName("buttonNum");
```

Comment:
These document.getElementsByClassName return arrays containing a single element, which is why your attempt to add an event listener to the element below is failing. You would have to select the first element in the array you get back from the query with [0]